src/analysis/retail/shaman/enhancement/modules/talents/LegacyOfTheFrostWitch.tsx

```diff
@@ -119,9 +125,11 @@ class LegacyOfTheFrostWitch extends Analyzer {
   }
 
   get extraDamage() {
-    return Object.keys(this.buffedSpells)
-      .map((guid) => this.buffedSpells[Number(guid)])
-      .reduce((current, total) => (total += current));
+    const spellList = Object.keys(this.buffedSpells).map((guid) => this.buffedSpells[Number(guid)]);
+    if (spellList?.length > 0) {
+      return spellList.reduce((current, total) => (total += current));
```

Comment:
`reduce` calls should provide a default argument to help avoid some fun bugs.

```suggestion
      return spellList.reduce((current, total) => (total += current), 0);
```